udata/search/__init__.py

```diff
@@ -129,18 +129,31 @@ def reindex(obj):
         log.info('Indexing %s (%s)', model.__name__, obj.id)
         try:
             adapter = adapter_class.from_model(obj)
-            adapter.save(using=es.client, index=es.index_name)
-        except:
-            log.exception('Unable to index %s "%s"',
-                          model.__name__, str(obj.id))
+            adapter.save(using=es.client, index=es.index_name, refresh=True)
```

Comment:
Isn't this a performance hog? https://github.com/elastic/elasticsearch-js/issues/117#issuecomment-48249941